.gitattributes

```diff
@@ -39,6 +41,8 @@ ocamlbuild/TODO          ocaml-typo=missing-header
 ocamldoc/Changes.txt     ocaml-typo=missing-header
 ocamldoc/ocamldoc.sty    ocaml-typo=missing-header
 
+patches/Patches.md       ocaml-typo=very-long-line,long-line,missing-header
```

Comment:
Line breaks are respected in Markdown (at least that seemed to be what was happening?) - if there's a way of escaping them (a la an end-of-line `%` in TeX) then great, but formatted with 80 character lines the rendered result had some very confusing line breaks.